docs/architecture/adr-0014-api-documentaton-revisited.md

```diff
@@ -0,0 +1,104 @@
+# API documentation ADR
+## Context
+
+The goal isn't to make this super complicated, but give us a basis for building out more 
+full-fledged documentation as the project goes -- beyond just listing API endpoints 
+(see Auth0's docs section as inspiration https://auth0.com/docs).
```

Comment:
The one that we had looked at is written in Backbone. I don't think it would be advantageous to piggyback on that.